storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/dataaccess/V3RocksDbDao.java

```diff
@@ -173,11 +174,33 @@ private void initialize() {
     }
 
     LOG.info("Initializing hot states from hot blocks");
-    initializeHotStates(finalizedRoot.get(), finalizedState.get(), hotBlocksByRoot);
+    final Map<Bytes32, SignedBeaconBlock> prunedHotBlocks =
+        initializeHotStatesAndBlocks(finalizedRoot.get(), finalizedState.get(), hotBlocksByRoot);
     LOG.info("Finished initializing hot states from hot blocks");
+
+    LOG.info("Adding hot roots to cache ordered by slots");
```

Comment:
This doesn't take a lot of time, so I don't think it's really worth logging the start / end of the cache initialization.

Reply:
makes sense to me.